rancher2/schema_node_template_hetznercloud.go

```diff
@@ -0,0 +1,69 @@
+package rancher2
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
```

Comment:
This import is not correct, please add correct schema import to build correctly

```
-	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
```

Reply:
@rawmind0, thanks for pointing this out. Fixed it and the CI build is now passing. Keep in mind though that I have not added any tests.

Reply:
@rawmind0, is there anything else you need from me to merge this change? I must admit that I'm fairly new to the PR process on github.